enhancements/ingress/optional-ingress-hypershift.md

```diff
@@ -0,0 +1,320 @@
+- [Ingress Config API](https://docs.openshift.com/container-platform/4.13/rest_api/config_apis/ingress-config-openshift-io-v1.html). Implemented in OpenShift API as [CRD](https://github.com/openshift/api/blob/master/config/v1/0000_10_config-operator_01_ingress.crd.yaml).
+- [Ingress API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#ingress-v1-networking-k8s-io). Implemented in Kubernetes.
+
+However the implementation of these APIs (or parts of them) depend on the ingress operator:
+- The ingress operator creates the default ingress controller to manage the backends for the routes. See [the drawbacks](#drawbacks) for further implications.
+  - The ingress objects are implicitly managed by the default ingress controller. See [the ingress to route controller risk](#ingress-to-route-controller) for further implications.
+- For [the component routes](https://docs.openshift.com/container-platform/4.13/rest_api/config_apis/ingress-config-openshift-io-v1.html#spec-componentroutes) the ingress operator creates the required role and role binding to get the custom TLS certificate secret from the `openshift-config` namespace. See [the component routes risk](#component-routes) for further implications.
+
+#### Console operator
+
+In case the ingress capability is not enabled in `ClusterVersion`, the following issues arise with the console components:
+- The console operator becomes degraded due to the failed admission checks of the console and download routes, see [the usage](https://github.com/search?q=repo%3Aopenshift%2Fconsole-operator%20IngressURI&type=code) of [IngressURI function](https://pkg.go.dev/github.com/openshift/library-go/pkg/route/routeapihelpers#IngressURI).
+- The console route is not served by the default ingress controller.
+
+To resolve the former problem, it is necessary to skip the route admission checks when the ingress capability is not enabled.   
+The later issue is discussed as a general case of the missing default ingress controller in [the drawbacks](#drawbacks).
+
+#### Authentication operator
+
+No change to the authentication operator is needed until the implementation of the ingress capability for the standalone OpenShift.
+HyperShift doesn't use the authentication operator, the authentication server is [managed by HyperShift's control plane directly](https://github.com/openshift/hypershift/blob/7cb87788779ae928d90d0e7760c1e9359f04e58a/control-plane-operator/controllers/hostedcontrolplane/oauth/deployment.go#L123-L127).
+
+#### Monitoring operator
+
+No change to the monitoring operator is needed until the implementation of the ingress capability for the standalone OpenShift.
+The monitoring operator skips the alert manager's route check if `ingress` cluster operator is not there.
```

Comment:
```suggestion
The monitoring operator skips the alert manager's route check if the `ingress` ClusterOperator CR is not present on the cluster.
```
Hm, why isn't the "ingress" clusteroperator present on a HyperShift hosted cluster?  It's in [the ingress operator's manifests](https://github.com/openshift/cluster-ingress-operator/blob/d319e461f1c74a80474804a447237a8a679c4abd/manifests/03-cluster-operator.yaml); [the CVO documentation](https://github.com/openshift/enhancements/blob/d09fbc431dcab82e730641900d5c97571b992153/dev-guide/cluster-version-operator/dev/clusteroperator.md) says, "ClusterVersionOperator will pre-create the ClusterOperator resource found in the `/manifests` folder"; and I don't see it in [`manifestsToOmit`](https://github.com/openshift/hypershift/blob/af2f0b13103f9ac1373b4abcd0b83a2870dac81c/control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go#L47-L98).